docs/user_guide/preemption.md

```diff
@@ -0,0 +1,249 @@
+---
+id: preemption_cases
+title: Preemption
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Preemption is an essential feature found in most schedulers, and it plays a crucial role in enabling key system functionalities like DaemonSets in K8s, as well as SLA and prioritization-based features.
+
+This document provides a brief introduction to the concepts and configuration methods of preemption in Yunikorn. For a more comprehensive understanding of Yunikorn's design and practical ideas related to preemption, please refer to the [design document](design/preemption.md).
```

Comment:
Nit: Change "Yunikorn" to "YuniKorn" (Y and K capitalized). There are multiple occurrences, all should be changed.